plugin/suda.vim

```diff
@@ -6,3 +6,21 @@ let g:loaded_suda = 1
 if get(g:, 'suda_startup', 1)
   call suda#init()
 endif
+
+if exists('g:suda_hijack_write')
+  cabbrev w W
+  cabbrev wa Wa
+  cabbrev wq Wq
+  cabbrev wqa Wqa
+  command! W execute suda#write_wrapper('')
+  command! Wa execute suda#write_wrapper('a')
+  command! Wq execute suda#write_wrapper('q')
+  command! Waq execute suda#write_wrapper('aq')
```

Comment:
**must**

You don' need `execute` here. Eg.

```vim
command! W suda#write_wrapper('')
```

Reply:
I think I am, I tried removing it and I got:

```
E492: Not an editor command: suda#write_wrapper('')
```

Reply:
My bad. use `call` like

```vim
command! W call suda#write_wrapper('')
```